docker/kratos/kratos.yml

```diff
@@ -63,6 +63,12 @@ selfservice:
                         emit_analytics_event: true
                         response:
                           ignore: false
+                        auth:
+                          type: api_key
+                          config:
+                            name: Authorization
+                            value: aba
```

Comment:
question: can we make it more like a uuid or something longer?

Reply:
Changed it to `secret_api_key`. This is the dev setup config, so it shouldn't really matter, in a real env it will be auto-generated by the `user-verification-service` charm